### PR TITLE
feat: LT-1027 - PV Grading

### PR DESCRIPTION
**Pull request by Mark-Spiers-kainos**

### CODE REVIEW

File: goods/views.py [hunk 1]
```diff
@@ -145,7 +148,7 @@ def post(self, request, **kwargs):
             if errors.get("text"):
                 error = errors.get("text")[0]
                 error = error.replace("This field", "Case note")
-                error = error.replace("this field", "the case note")  # TODO: Move to API
```
Comment (jfaracik) [inline goods/views.py]: Why was this comment removed?

Comment (Mark-Spiers-kainos) [reply goods/views.py]: honestly, I we don't remember removing it, added back in.

File: applications/views/goods.py [hunk 2]
```diff
@@ -78,12 +81,23 @@ def get(self, request, **kwargs):
...
     def init(self, request, **kwargs):
-        self.draft_pk = kwargs["pk"]
-        self.form = add_goods_questions(self.draft_pk)
+        self.draft_pk = str(kwargs["pk"])
+        self.forms = add_good_form_group(draft_pk=self.draft_pk)
         self.action = post_goods
 
+    def on_submission(self, request, **kwargs):
+        self.draft_pk = str(kwargs["pk"])
+        pv_grading = request.POST.copy().get("is_pv_graded", "").lower() == "yes"
```
Comment (stephanieboyd) [inline applications/views/goods.py]: Please change this as discussed

File: assets/css/components/_pv-grading.scss [hunk 3]
```diff
@@ -0,0 +1,7 @@
+.app-pv-grading-inputs {
+	@include govuk-media-query($from: desktop) {
+		display: grid;
+		grid-template-columns: 1fr 1fr 1fr!important;
+		grid-gap: govuk-spacing(6);
+	}
+}
```
Comment (stephanieboyd) [inline assets/css/components/_pv-grading.scss]: Can you check your edited .scss files for a newline at the end? We include those

File: goods/forms.py [hunk 4]
```diff
@@ -190,7 +232,43 @@ def attach_documents_form(back_url):
             ),
         ],
         buttons=[Button(AttachDocumentForm.BUTTON, "submit", disable_double_click=True)],
-        back_link=BackLink(AttachDocumentForm.BACK_LINK, back_url),
+        back_link=back_link,
+    )
+
+
+def raise_a_pv_or_clc_query(good_id, raise_a_clc: bool, raise_a_pv: bool):
```
Comment (stephanieboyd) [inline goods/forms.py]: `raise_a_goods_query`

File: goods/services.py [hunk 5]
```diff
@@ -24,36 +23,99 @@ def get_good(request, pk):
 
 
 def post_goods(request, json):
-    if json.get("good_description", False) or json.get("good_description") == "":
-        post_data = remove_prefix(json, "good_")
-    else:
-        post_data = json
+    # if json.get("good_description", False) or json.get("good_description") == "":
```
Comment (stephanieboyd) [inline goods/services.py]: Can we delete this?

File: goods/services.py [hunk 6]
```diff
@@ -24,36 +23,99 @@ def get_good(request, pk):
...
+    #     post_data = remove_prefix(json, "good_")
+    # else:
+    post_data = json
+
     data = post(request, GOODS_URL, post_data)
+
+    if data.status_code == 200:
```
Comment (stephanieboyd) [inline goods/services.py]: Check for the code how we usually do please

**Pull request merged**